content/applications/hr/attendances/kiosks.rst

```diff
@@ -0,0 +1,187 @@
+======
+Kiosks
+======
+
+Odoo's *Attendances* app allows employees to check in and out of work directly from the database, or
+from a kiosk.
+
+Only employees with access to the database can check in and out from the database, and they are
+referred to as *users*. Kiosks are needed for employees who do **not** have access to the database.
+
+A kiosk is a dedicated device (a PC, tablet, or mobile phone) for employees to use when they check
+in and out.
+
+.. important::
+   If employees check in and out using a badge or an RFID, then an accessible device in *Kiosk Mode*
+   **must** be available in order to use these two methods.
+
+Configuration
+=============
+
+There are only a few configurations needed to use kiosks in the *Attendances* application. Navigate
+to :menuselection:`Attendances app --> Configuration` to access the :guilabel:`Settings` page.
+
+Once all desired settings have been configured, click the :guilabel:`Save` button on the
+:guilabel:`Settings` page, to activate and enable them.
+
+.. _attendances/kiosk-mode:
+
+Kiosk Mode section
+------------------
+
+Using the drop-down menu, select how an employee checks in when using a kiosk. Options are
+:guilabel:`Barcode/RFID`, :guilabel:`Barcode/RFID and Manual Selection`, or :guilabel:`Manual
+Selection`.
+
+.. important::
+   The *Barcode* application **does not** need to be installed to use one of the Barcode/RFID
+   settings.
+
+.. _attendances/kiosk-settings:
+
+Kiosk Settings section
+----------------------
+
+The various settings in the :guilabel:`Kiosk Settings` section determine how employees check in and
+out with kiosks.
+
+- :guilabel:`Barcode Source`: this setting **only** appears if one of the two Barcode/RFID
+  selections were configured for the :ref:`Kiosk Mode <attendances/kiosk-mode>` setting.
+
+  If available, select how barcodes are scanned at the kiosk, via one of the drop-down menu options.
+  Barcodes can be scanned with a dedicated :guilabel:`Scanner`, or with a device's camera
+  (:guilabel:`Front Camera` or :guilabel:`Back Camera`).
+- :guilabel:`Employee PIN Identification`: tick this checkbox if employees should use a unique PIN
+  to check in. PINs are configured on each individual employee record. Refer to the
+  :doc:`../employees/new_employee` documentation for more information on setting up PINs.
```

Comment:
Link directly to the relevant section, since it is available :)
```suggestion
  :ref:`new employee documentation <employees/hr-settings>` for more information on setting up PINs.
```